code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -264,4 +264,38 @@
 	name = "smashed butter"
 	desc = "Looks like some one has butter fingers."
 	icon = 'icons/effects/tomatodecal.dmi'
-	icon_state = "smashed_butter"
\ No newline at end of file
+	icon_state = "smashed_butter"
+
+/obj/effect/decal/cleanable/virusdish
+	name = "broken virus containment dish"
+	icon = 'icons/obj/virology.dmi'
+	icon_state = "brokendish-outline"
+	density = 0
+	anchored = 1
+	mouse_opacity = 1
+	layer = OBJ_LAYER
+	plane = OBJ_PLANE
+	var/last_openner
```

Comment:
this variable is never set

Reply:
it gets set in items_devices.dm 
![image](https://user-images.githubusercontent.com/7573912/60885344-aa03c280-a24f-11e9-823e-136f4d5e5bbb.png)

Reply:
Ah, my bad.